packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -129,7 +127,9 @@ export function useBlockProps(
 			enableAnimation,
 			triggerAnimationOnChange: index,
 		} ),
-		useDisabled( { isDisabled: ! __unstableIsDisabled } ),
+		useDisabled( {
+			isDisabled: ! hasOverlay,
```

Comment:
Nice one to move this inside and reads so much better than `{ isDisabled: ! __unstableIsDisabled }`